audit-base/src/main/java/com/wultra/core/audit/base/database/DatabaseAuditWriter.java

```diff
@@ -0,0 +1,244 @@
+/*
+ * Copyright 2021 Wultra s.r.o.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.wultra.core.audit.base.database;
+
+import com.wultra.core.audit.base.AuditWriter;
+import com.wultra.core.audit.base.configuration.AuditConfiguration;
+import com.wultra.core.audit.base.model.AuditRecord;
+import com.wultra.core.audit.base.util.ClassUtil;
+import com.wultra.core.audit.base.util.JsonUtil;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.jdbc.core.BatchPreparedStatementSetter;
+import org.springframework.jdbc.core.JdbcTemplate;
+import org.springframework.jdbc.core.PreparedStatementCallback;
+import org.springframework.scheduling.annotation.Scheduled;
+import org.springframework.stereotype.Service;
+
+import javax.annotation.PreDestroy;
+import java.io.PrintWriter;
+import java.io.StringWriter;
+import java.sql.*;
+import java.time.Instant;
+import java.time.LocalDateTime;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.concurrent.BlockingQueue;
+import java.util.concurrent.LinkedBlockingDeque;
+
+/**
+ * Database audit writer.
+ *
+ * @author Roman Strobl, roman.strobl@wultra.com
+ */
+@Service
+public class DatabaseAuditWriter implements AuditWriter {
+
+    private static final Logger logger = LoggerFactory.getLogger(DatabaseAuditWriter.class);
+
+    private static final String SQL_INSERT_INTO = "INSERT INTO ";
```

Comment:
I am not sure about this. Does it make the code more readable?

Reply:
Added.

Reply:
Ok, we can change it back.